Modules/Sources/Support/UI/Support Conversations/SupportForm.swift

```diff
@@ -0,0 +1,394 @@
+import Foundation
+import SwiftUI
+import PhotosUI
+
+public struct SupportForm: View {
```

Comment:
This view does not have a cancel button. Even though we can swipe down to dismiss it, I think the app usually still shows an X or "Close" button on similar screens.